modules/component-store/schematics-core/tsconfig.lib.json

```diff
@@ -1,18 +1,22 @@
 {
-  "extends": "./tsconfig.json",
```

Comment:
I think this config was initially wrong, (tsconfig references tsconfig.lib.json, but tsconfig.lib.json extended tsconfig.json). This change, keeps the config aligned with the config of the other modules.